demos/common/python/models/retinaface.py

```diff
@@ -302,3 +303,153 @@ def landmark_pred(boxes, landmark_deltas):
             pred[:, i, 1] = landmark_deltas[:, i, 1] * heights + ctr_y
 
         return pred
+
+
+class RetinaFacePyTorch(Model):
+    def __init__(self, ie, model_path, threshold=0.5):
+        super().__init__(ie, model_path)
+
+        assert len(self.net.input_info) == 1, "Expected 1 input blob"
+        expected_outputs_count = (2, 3)
+        assert len(self.net.outputs) in expected_outputs_count, "Expected {} or {} output blobs".format(
+            expected_outputs_count[0], expected_outputs_count[1])
+
+        self.threshold = threshold
+        self.process_landmarks = len(self.net.outputs) == 3
+        self.postprocessor = RetinaFacePyTorchPostprocessor(process_landmarks=self.process_landmarks)
+
+        self.labels = ['Face']
+
+        self.image_blob_name = next(iter(self.net.input_info))
+        self._output_layer_names = self.net.outputs
+        self.n, self.c, self.h, self.w = self.net.input_info[self.image_blob_name].input_data.shape
+
+    def preprocess(self, inputs):
+        image = inputs
+
+        resized_image = resize_image(image, (self.w, self.h))
+        meta = {'original_shape': image.shape,
+                'resized_shape': resized_image.shape}
+        resized_image = resized_image.transpose((2, 0, 1))  # Change data layout from HWC to CHW
+        resized_image = resized_image.reshape((self.n, self.c, self.h, self.w))
```

Comment:
please use expand_dims instead (reshaping images to the same shape but with adding batch dimension looks reductant)

Reply:
Done